modules/govuk/lib/facter/fqdn_metrics.rb

```diff
@@ -7,7 +7,7 @@
     if Facter.value("aws_migration").nil?
       fqdn_metrics = Facter.value("fqdn_short").dup
     else
-      fqdn_metrics = "#{Facter.value(:aws_hostname)}.#{Facter.value(:aws_stackname)}.#{Facter.value(:aws_environment)}"
+      fqdn_metrics = "#{Facter.value(:aws_migration)}-#{Facter.value('fqdn')}"
```

Comment:
Given the warning at the top of the file about the 63 byte limit, I am unsure if this is going to be ok?

As far as I'm aware, the current collectd namespaces use this pattern, and one long one I pulled out comes to exactly 63 characters by my count (`rummager_elasticsearch-ip-10-1-4-149_eu-west-1_compute_internal`).

Reply:
ah yeah didn't think of that, on AWS staging we have rummager_elasticsearch-ip-10-2-4-246_eu-west-1_compute_internal which is 64 characters long, and it seems to be collecting normally though

Reply:
Looks from the 5.6 changelog that this is now configurable (look for DATA_MAX_NAME_LEN), and might be longer by default, but we are using 5.4 as far as I can see...

it would be good to confirm that this is no longer a problem if it really isn't, and remove the comment if that's the case.

Reply:
I see this has changed on Friday, I think it's fine to try using the more recent version of collectd.